tensorflow/lite/kernels/range.cc

```diff
@@ -88,21 +88,18 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
   // Currently only supports int32 and float.
   // TODO(b/117912892): Support quantization as well.
   const auto dtype = start->type;
-  TF_LITE_ENSURE(context, dtype == kTfLiteInt32 || dtype == kTfLiteFloat32);
   TF_LITE_ENSURE_EQ(context, limit->type, dtype);
   TF_LITE_ENSURE_EQ(context, delta->type, dtype);
   TfLiteTensor* output = GetOutput(context, node, kOutputTensor);
 
   switch (dtype) {
```

Comment:
nit: can you do:

 if not supported type:
   report error and return;

following code...

Reply:
@renjie-liu : Sorry if i am not clear about your comment here. As per i understood, the behavior you mentioned is exactly covered in the switch case below right? Please correct me if i am wrong, TIA!

Reply:
I was suggesting an early return using if:

if (dtype != kTfLiteFloat32 || dtype != kTfLiteInt32) {
  // Report error.
  return kTfLiteError;
}
output_type = dtype;